### ISSUE TITLE
Temp, Age movies

### ISSUE THREAD

**Issue by PaulSpence** [adele157/easterlies-collaborative-project#33]
Create movies of temp and bottom age anomalies on the shelf over first few years.

**Comment by julia-neme** [comment]
Some animations for the first 5 years of the experiment. They are all monthly anomalies with respect to CONTROL. 

## Bottom temperature  

https://user-images.githubusercontent.com/51519847/131603789-8b3aaa66-625a-423e-a66d-9f287be086d6.mp4

## Bottom salinity

https://user-images.githubusercontent.com/51519847/131603901-790ada1c-bcaf-4b37-910f-67c37baa276e.mp4

## Bottom age

https://user-images.githubusercontent.com/51519847/131603915-37682e12-6650-474a-ab4a-57ed109ca0d7.mp4

**Comment by AndyHoggANU** [comment]
For some reason, I can't view these animations. Anyone else in the same boat?

**Comment by adele157** [comment]
I can see them if I go to the Issue.

**Comment by AndyHoggANU** [comment]
Nope, doesn't help. Maybe it is because you are the owner ??

**Comment by StephenGriffies** [comment]
I see upside down smiley faces...  Must be an ownership issue.

**Comment by adele157** [comment]
I think it's a browser [compatibility](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files) issue. I can see them in chrome, but not safari and not on my phone.

**Comment by julia-neme** [comment]
I've edited to make the encoding h264, can you check if you are able to see them now? Otherwise, just to test, maybe .gif format is better? Although you won't be able to pause and go forward or back.

![btemp-monthly-5year](https://user-images.githubusercontent.com/51519847/131762601-86fbae88-45e1-432b-a133-bae6bdf67f6f.gif)

**Comment by StephenGriffies** [comment]
Yep, as @adele157 said, I can see the original videos using Chrome but not Firefox.  Thanks also to @julia-neme for the gifs, which work in Firefox.

**Comment by StephenGriffies** [comment]
Amazing videos.  Love the coastal fjord signals. 

Lovely temp signal very close to the coast especially on the narrow shelf regions to west of Ross moving towards Weddell.  Must be advection?   

Is the cool region to the west of the Ross in the UP expt one of the katabatic wind spots where bottom water forms?   

Bottom salinity signal is more diffuse than temp; quite the opposite what we see in surface anomalies.  

Can we explain the two regions of young age (up and downstream of Ross) in the UP experiment via katabatic wind changes?

**Comment by AndyHoggANU** [comment]
Thanks Julia -- yep, I can see these now.
I agree, it is weird that salinity is diffuse while temperature anomalies are confined to the fjords. Also, the temperature anomaly isn't symmetric with respect to the perturbation -- DOWN acts to warm bottom temperature, while UP initially cools in many places, then later on warms in most places. This doesn't seem to me to be a straightforward response ...

**Comment by StephenGriffies** [comment]
Yes.  For example, the western side of the peninsula is mostly warming in both, though more muted in the UP.

**Comment by adele157** [comment]
@StephenGriffies regarding "Is the cool region to the west of the Ross in the UP expt one of the katabatic wind spots where bottom water forms?"

No, I've circled the Adelie bottom water region here:
<img width="323" alt="Screen Shot 2021-09-03 at 5 07 14 am" src="https://user-images.githubusercontent.com/8506963/131902347-a9fa6f7d-cc79-4d5a-aca7-6fd201b40888.png">

**Comment by StephenGriffies** [comment]
So is that consistent with not seeing much changes in overall ventilation when cut off the katabatic winds?  Namely, the katabatics do not seem to be locally affecting bottom water formation.  Do you agree?

**Comment by matthew-england-unsw** [comment]
I see it more as the katabatic winds not being the limiting factor here.   So in the experiment where they remain unchanged locally, DSW is still formed.

**Comment by matthew-england-unsw** [comment]
I suspect if they were truly zeroed out in the experiment we’d see DSW formation significantly reduced.   Those latest experiments merely suppressed any 10% increase there, while winds were amplified everywhere else.  So I’m thinking the baseline katabatics still play a role.

**Comment by StephenGriffies** [comment]
Ah yea, of course. I was thinking the winds were set to zero there.  Good point.
